library/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.chuckerteam.chucker">
-    <application>
+    <application
+      android:supportsRtl="true">
```

Comment:
This should be avoided. Ideally we should not set a `android:supportsRtl` in the library as this gets merged in the final manifest. The default value for `supportsRtl` is false.

Moreover we might break builds for users that have `supportsRtl` set to `false` on the host app. See also https://github.com/ChuckerTeam/chucker/pull/121#issuecomment-533480441